src/test/java/nl/knaw/dans/ingest/core/service/mapper/MappingIntegrationTest.java

```diff
@@ -0,0 +1,145 @@
+    void setUp() {
+        iso1ToDataverseLanguage.put("nl", "Dutch");
+        iso1ToDataverseLanguage.put("de", "German");
+
+        iso2ToDataverseLanguage.put("dut", "Dutch");
+        iso2ToDataverseLanguage.put("ger", "German");
+    }
+
+    @Test
+    void DD_1216_description_type_other_maps_only_to_author_name() throws Exception {
+        var doc = readDocumentFromString(
+            "<ddm:DDM xmlns:ddm=\"http://easy.dans.knaw.nl/schemas/md/ddm/\"\n"
+            + "         xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
+            + "         xmlns:dc=\"http://purl.org/dc/elements/1.1/\"\n"
+            + "         xmlns:dct=\"http://purl.org/dc/terms/\"\n"
+            + "         xsi:schemaLocation=\"http://easy.dans.knaw.nl/schemas/md/ddm/ http://easy.dans.knaw.nl/schemas/md/2017/09/ddm.xsd\">\n"
+            + ddmProfile
+            + "    <ddm:dcmiMetadata>\n"
+            + "        <dct:rightsHolder>Mr. Rights</dct:rightsHolder>\n"
+            + "        <dct:description descriptionType=\"Other\">Author from description other</dct:description>\n"
+            + "    </ddm:dcmiMetadata>\n"
+            + "</ddm:DDM>\n");
+
+        var result = mapDdmToDataset(doc);
+        var field = (CompoundField) result.getDatasetVersion().getMetadataBlocks()
+            .get("citation").getFields().stream()
+            .filter(f -> f.getTypeName().equals("contributor")).findFirst().orElseThrow();
+        var expected = "Author from description other";
+        assertThat(field.getValue())
+            .extracting(AUTHOR_NAME)
+            .extracting("value")
+            .containsOnly(expected);
+        assertEquals(2, toJsonString(result).split(expected).length);
+    }
+
+    @Test
+    void DD_1216_description_type_technical_info_maps_once_to_description() throws Exception {
+        var doc = readDocumentFromString(
+            "<ddm:DDM xmlns:ddm=\"http://easy.dans.knaw.nl/schemas/md/ddm/\"\n"
+                + "         xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
+                + "         xmlns:dc=\"http://purl.org/dc/elements/1.1/\"\n"
+                + "         xmlns:dct=\"http://purl.org/dc/terms/\"\n"
+                + "         xsi:schemaLocation=\"http://easy.dans.knaw.nl/schemas/md/ddm/ http://easy.dans.knaw.nl/schemas/md/2017/09/ddm.xsd\">\n"
+                + ddmProfile
+                + "    <ddm:dcmiMetadata>\n"
+                + "        <dct:rightsHolder>Mr. Rights</dct:rightsHolder>\n"
+                + "        <dct:description descriptionType=\"TechnicalInfo\">technical description</dct:description>\n"
+                + "        <dct:description descriptionType=\"NotKnown\">not known description type</dct:description>\n"
+                + "    </ddm:dcmiMetadata>\n"
+                + "</ddm:DDM>\n");
+
+        var result = mapDdmToDataset(doc);
+        var str = toJsonString(result);
+        assertFalse(str.contains("not known description type"));
```

Comment:
Not sure this is the desired interpretation of the spec. Perhaps not otherwise mapped types must be converted to a description.

Reply:
Tried series shown with [PR](https://github.com/DANS-KNAW-jp/dd-ingest-flow/pull/1/) on this PR, despite multiple==false we get [...] as documented with todo in test class

Reply:
discussed with @lindareijnhoudt indeed map not otherwise mapped descriptions